Shared/UI/Views/SpeechBubbleView.swift

```diff
@@ -64,7 +64,7 @@ class SpeechBubbleView: BaseView {
 
         super.init()
 
-        self.setBubbleColor(ThemeColor.B1.color, animated: false)
+        self.setBubbleColor(bubbleColor, animated: false)
```

Comment:
Minor bug in our speech bubble view. It didn't actually respect the initial color you pass in.